config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root 'tasks#index'
+  resources :tasks, only: [:index, :create, :update, :destroy]
+end
```

Comment:
Does this follow the requirements in the readme?

|VERB   |PATH   |User for   |
|---|---|---|
|GET |/tasks |display a list of all tasks
|POST  |/tasks |create a new task
|GET |/tasks/:id |display a specific task
|PATCH/PUT |/tasks/:id |update a specific task
|DELETE  |/tasks/:id |delete a specific task